azure-pipelines.yml

```
@@ -32,7 +32,13 @@ jobs:
       ## manually call this since lage is now used and it calls lint per package, not per repo
       - script: |
           yarn prelint
-          yarn prettier $(targetBranch) --check
+
+          if [ $BUILD_SOURCEBRANCH == "refs/heads/master" ]; then
```

why not just check the existence of the `$(targetBranch)` in the `if` condition ?

---

¯\_(ツ)_/¯

It's also an option 👍

```suggestion
          if [ -n "$TARGETBRANCH" ]; then
```

Will it be like this?

---

Yeah that should be it, since we want to avoid duplicating the `branch == master` condition